src/Peering/Peering/help/New-AzPeeringDirectConnectionObject.md

```diff
@@ -12,26 +12,19 @@ Creates a in memory PSObject to be used for creating or modifying a Peering.
 
 ## SYNTAX
 
-### IPv4Prefix (Default)
+### IPv4PrefixIPv6Prefix (Default)
 ```
-New-AzPeeringDirectConnectionObject [-PeeringDBFacilityId] <Int32> [-SessionPrefixV4] <String>
- [-BandwidthInMbps] <Int32> [-MaxPrefixesAdvertisedIPv4 <Int32>] [-MD5AuthenticationKey <String>]
+New-AzPeeringDirectConnectionObject [-PeeringDBFacilityId] <Int32> [-SessionPrefixV4 <String>]
+ [-SessionPrefixV6 <String>] -BandwidthInMbps <Int32> [-UseForPeeringService]
```

Comment:
@rileymckenna is it intended that `-BandwidthInMbps` is no longer positional?

Reply:
yes as it may or may not have the switch parameter